docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md

```diff
@@ -258,6 +261,62 @@ One major benefit that we get from doing this is that we know the building of th
 
 Since these images will contain code from within a private repo, we need to make sure that unauthenticated users do not have access to these images. We prevent this by adding an [RBAC file](https://github.com/openshift/release/pull/36522/files#diff-e129896e392bc9ced499cb19ddf549b361784048825b25670755fc432f87f542) to only allow authenticated users to pull from the target registry namespace.
 
+### Mirror Quay Images
+
+There might be a case when we need to use the existing quay image in our scenario. The image can be in either public or private registry. For this purpose, Openshift CI allows to use external images by mirroring them into Openshift CI registry. Create separate pull request to mirror image in Openshift CI and get it merged before using that image in any job or pull request rehearsals. Once the image is mirrored in Openshift CI registry, use it as `base_image` in config file.
+
+#### How to Mirror Public Image
+
+If the image is public, we can mirror the image by adding a mapping in file `core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci` of [openshift/release](https://github.com/openshift/release/blob/master/core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci) repository like this:
+
+```yaml
+quay.io/QUAY_REPOSITORY/test-image:latest registry.ci.openshift.org/ci/test-image:latest
```

Comment:
you can add the format of mapping here, and then the example

Reply:
@Simranpal example is already added in L273 and for example I've added AMQ Streams PR in L276. Can you elaborate what other example you want me to add here?

Reply:
The existing example is fine. I meant the format and example to be together something like this:

```
Format of image mapping: quay.io/QUAY_REPOSITORY/IMAGE_NAME:latest registry.ci.openshift.org/$FOLDER_NAME/IMAGE_NAME:latest

An example for 3scale image mapping : quay.io/3scale/test-image:latest registry.ci.openshift.org/3scale-qe/test-image:latest
```
The example may need to be corrected. If the example is in other PR, please ignore this comment

Reply:
I have updated the above line L270 to make it clear the format is provided and example is in the PR. Please review the changes @Simranpal

Reply:
alright, thanks!